File: beecrowd/beginner/c/1118.c
#include <stdio.h>

int main() {
    float score, total_score = 0;
    int x = 0, count = 0;

    while (1) {
        scanf("%f", &score);

        if (score >= 0 && score <= 10) {
            total_score += score;
            count += 1;

            if (count == 2) {
                printf("media = %.2f\n", total_score / 2);
            }
        } else {
            printf("nota invalida\n");
            continue;
        }

        while (count == 2 && (x > 2 || x < 1)) {
            scanf("%d", &x);
            printf("novo calculo (1-sim 2-nao)\n");
        }

        if (x == 2) break;
        if (x == 1) total_score = count = x = 0;      
        
    }

    
    return 0;
}

File: beecrowd/beginner/python/1118.py
sum = count = 0

while 1:
    n = float(input())

    if 0 <= n <= 10:
        sum += n
        count += 1
    else:
        print("nota invalida")

    if count == 2:
        print(f"media = {sum / 2:.2f}")

        while 1:
            print("novo calculo (1-sim 2-nao)")
            op = int(input())

            if op == 2:
                exit()
            elif op == 1:
                break
        sum = count = 0
